scp/templates/bootstrap/bootstrap-job-secret.yaml

```diff
@@ -0,0 +1,22 @@
+{{/*
+  Secret for keycloak Bootstrapper - do not create default users
+  */}}
+{{- if .Values.bootstrap.keycloak }}
+{{- if .Values.secrets.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "common.lib.fullname" . }}-keycloakbootstrap-secret
+  labels:
+    {{- include "common.lib.labels" . | nindent 4 }}
+type: Opaque
+stringData:
+  CLIENT_SECRET: {{ .Values.secrets.keycloakBootstrap.clientSecret }}
```

Comment:
There is only one client secret here, are we only installing one client?  I thought we were doing one client per PEP?

Reply:
This setting here is only for the hard-wired clients the bootstrapper creates now in OTDF.

All of the custom clients can be configured via the clients list ref'd [here](https://github.com/virtru-corp/virtru-charts/blob/add-config-to-helm-repo_bootstrapping/scp/templates/bootstrap/bootstrap-config-secret.yaml#L18): 

Some brief description in this is [here](https://github.com/virtru-corp/virtru-charts/blob/add-config-to-helm-repo_bootstrapping/scp/README.md#bootstrapping-deployment):